cypress/e2e/signin_with_totp/index.cy.ts

```diff
@@ -36,6 +36,27 @@ describe("sign-in with TOTP on untrusted browser", () => {
     cy.contains('"amr": [\n    "pwd",\n    "totp",\n    "mfa"\n  ],');
   });
 
+  it("should sign-in with TOTP when forced by SP, password only otherwise", function () {
```

Comment:
issue: this test looks like a concatenation of the too previous tests. As such, it's seems redundant. Plus, the feature implemented in this PR is not tested. We should have a test where the user do not tick the "force_2fa" when enrolling is totp and is not asked for totp on is next connexion. Also we need a test on the scenario where the user tick the box.